app/models/notebook.rb

```diff
@@ -474,7 +474,7 @@ def snippet(user)
     recommendations = recommendation_snippet
     snippet =
       if highlights && recommendations
-        "#{highlights}<br><br>#{recommendations}"
```

Comment:
Rather than just remove the <br>, find examples that actually fill out those two and make sure it still displays correct.  Or rather than just saying if x&&y check if they are also non-zero length as well.  This feels like a lazy hack that may cause other display issues.